web/app.js

```diff
@@ -1457,6 +1460,9 @@ function webViewerInitialized() {
     if ('useonlycsszoom' in hashParams) {
       PDFJS.useOnlyCssZoom = (hashParams['useonlycsszoom'] === 'true');
     }
+    if ('printautorotate' in hashParams) {
```

Comment:
> How would you test the parameter in the web viewer without the hash param?

E.g. by executing `PDFViewerApplication.preferences.set('enablePrintAutoRotate', {true, false});` in the console and then reloading the viewer.

Furthermore, we shouldn't be adding new things to `PDFJS` *especially* not in the viewer, since the usage of `PDFJS` will most likely be removed in future PDF.js versions.
Can we please just pass the preference into `PDFViewer` in the same way as e.g. the `enhanceTextSelection` preference is handled instead!?